internal/lib/stats/descriptors.go

```diff
@@ -234,4 +234,9 @@ var (
 		Help:      "Memory swap limit for the container.",
 		LabelKeys: baseLabelKeys,
 	}
+	containerStartTimeSeconds = &types.MetricDescriptor{
```

Comment:
Do we have to add this metric to:

https://github.com/cri-o/cri-o/blob/8e164e8b78ce4e44302259e946acce816d287ae0/internal/lib/stats/metrics.go#L108-L115

Reply:
great catch! updated.